packages/webpack-cli/lib/plugins/InteractiveModePlugin.js

```diff
@@ -0,0 +1,242 @@
+const readline = require('readline');
+const { red, green, cyanBright, bold } = require('colorette');
+const logger = require('../utils/logger');
```

Comment:
For plugin please use `compiler.getInfrastructureLogger` https://webpack.js.org/api/logging/#examples-of-how-to-get-and-use-webpack-logger-in-loaders-and-plugins